client/src/Components/Search/index.js

```diff
@@ -0,0 +1,53 @@
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+import { SearchOutlined } from '@ant-design/icons';
+import Button from '../Button';
+import Select from '../Select';
+
+import './style.css';
+
+import { locations, serviceTypes } from '../../Utils/data';
+
+const Filter = ({ handleSearch }) => {
+  const [service, setService] = useState('null');
```

Comment:
if you want to set a null value to any state keep it empty ``` useState(); ``` or ``` useState(null) ``` in this way you set a string, not a null value